kahuna/package.json

```diff
@@ -52,13 +52,14 @@
     }
   },
   "devDependencies": {
+    "htmllint-cli": "0.0.4",
     "jscs": "^1.13.0",
     "jshint": "^2.5.11",
     "jspm": "~0.16.0"
   },
   "scripts": {
     "test": "npm run lint",
-    "lint": "jscs public/js/ && jshint public/js --exclude public/js/dist/build.js",
+    "lint": "jscs public/js/ && jshint public/js --exclude public/js/dist/build.js && htmllint public/js/**/*.html",
```

Comment:
Good point, it wouldn't check in the directives. We seem to be doing mostly separate templates but worth keeping in mind..